test/test_dbcore.py

```diff
@@ -760,8 +762,31 @@ def test_no_results(self):
             ModelFixture1, dbcore.query.FalseQuery()).get())
 
 
+class ParentalDirCreation(unittest.TestCase):
+    @mock.patch('builtins.input', side_effect=['y', ])
+    def test_create_yes(self, _):
+        non_exist_path = "ParentalDirCreationTest/nonexist/" + str(random())
```

Comment:
I think this should be using a path within a temporary directory, to be certain that we don't pollute the environment where the test is run.